thoth/storages/graph/postgres.py

```diff
@@ -406,6 +406,26 @@ def get_alembic_version_count_all(self) -> int:
             result = session.execute(query).fetchone()
             return result[0]
 
+    def get_last_solver_datetime(
+        self, os_name: Optional[str] = None, os_version: Optional[str] = None, python_version: Optional[str] = None
+    ) -> datetime:
```

Comment:
I tested locally and this is a `string` actually, we need to modify the output if you expect to be of type `datetime`

Reply:
Right, thanks!

Reply:
But if this method is going to be used in metrics-exporter, string is totally fine :)

Reply:
If it's ok with you, we can leave it as a `datetime` just in case we reuse the method somewhere else, and I can eventually make changes later if it is more practical to have a string :)

Reply:
fine for me, but I'm not mistaken we will need to reconvert it to string to use it as label in a prometheus metric :)